waspc/src/Wasp/Generator/ServerGenerator/ConfigG.hs

```diff
@@ -22,8 +22,9 @@ genConfigFile spec = return $ C.mkTmplFdWithDstAndData tmplFile dstFile (Just tm
     tmplData =
       object
         [ "isAuthEnabled" .= isAuthEnabled spec,
-          "databaseUrlEnvVarName" .= databaseUrlEnvVarName
+          "databaseUrlEnvVarName" .= databaseUrlEnvVarName,
+          "defaultClientUrl" .= WebApp.getDefaultClientUrl spec
         ]
 
-configFileInSrcDir :: Path' (Rel C.ServerSrcDir) File'
-configFileInSrcDir = [relfile|config.js|]
+    configFileInSrcDir :: Path' (Rel C.ServerSrcDir) File'
```

Comment:
I saw that it was used only in one place, therefore I moved it inside of the `genConfigFile`

Reply:
I would say it is not about how it is used currently, it is about how it is intended to be used. We keep all of such paths at top level (if I am correct), because they are often by other modules for imports and similar. You putting into `where` indicates you mean it to be private/local for this function, so if I want to use it later, I will have to wonder if I can use that one -> it seems like the author didn't intend so.

So I would leave where it was.

Reply:
Makes sense 👍